Examples/src/main/java/com/mapbox/vision/examples/ArActivity.java

```diff
@@ -344,4 +345,46 @@ public void userOffRoute(Location location) {
     private List<Point> buildStepPointsFromGeometry(String geometry) {
         return PolylineUtils.decode(geometry, Constants.PRECISION_6);
     }
+
+    private ManeuverType mapToManeuverType(@Nullable String maneuver) {
+        if (maneuver == null) {
+            return ManeuverType.None;
+        }
+        switch (maneuver) {
```

Comment:
is it possible to create map String to Type?

Reply:
I guess it'll be nicer, lets do it!

Reply:
BUT WHY

Reply:
WHY KIRYL 
![image](https://user-images.githubusercontent.com/2265924/73379176-defe6200-42d2-11ea-9155-f5c013071821.png)

Reply:
Reverted back to `switch`.